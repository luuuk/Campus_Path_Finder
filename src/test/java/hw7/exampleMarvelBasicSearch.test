# Load the Marvel dataset and search for an existing path.
# To run this test, you will need to have downloaded the marvel dataset and
# place it in src/test/resources/data/hw7/marvel.tsv.

LoadGraph marvel marvel.tsv

FindPath marvel OLD_SKULL PHARAOH_RAMA-TUT
