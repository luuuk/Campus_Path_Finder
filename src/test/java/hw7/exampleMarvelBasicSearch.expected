# Load the Marvel dataset and search for an existing path.
# To run this test, you will need to have downloaded the marvel dataset and
# place it in src/test/resources/data/hw7/marvel.tsv.

loaded graph marvel

path from OLD SKULL to PHARAOH RAMA-TUT:
OLD SKULL to HAWK with weight 0.077
HAWK to CAPTAIN AMERICA with weight 0.003
CAPTAIN AMERICA to PHARAOH RAMA-TUT with weight 0.019
total cost: 0.099
